lib/Cake/Network/Email/CakeEmail.php

```diff
@@ -1203,7 +1203,7 @@ public function send($content = null) {
  * @throws SocketException
  */
 	public static function deliver($to = null, $subject = null, $message = null, $transportConfig = 'fast', $send = true) {
-		$class = __CLASS__;
+		$class = get_called_class();
```

Comment:
@stickler-ci In spite of your mustache you are too young to know that Cake 2.x uses tabs.